news-service/src/main/java/ru/clevertec/service/impl/NewsService.java

```
@@ -61,4 +67,12 @@ public boolean delete(Long id) {
                 })
                 .orElse(false);
     }
+
+    public List<NewsReadDto> findAll(NewsFilter filter, Pageable pageable) {
+        if (filter == null) {
+            return this.findAll();
```

я бы убрал эту проверку и просто пусть падало бы исключение - т.е. использовать подход fail fast

---

убрал